docs/operations/containerized_data_importer.md

```diff
@@ -21,7 +21,7 @@ virtctl in your path.
 Install the latest CDI release
 [here](https://github.com/kubevirt/containerized-data-importer/releases)
 
-    VERSION=$(curl -s https://github.com/kubevirt/containerized-data-importer/releases/latest | grep -o "v[0-9]\.[0-9]*\.[0-9]*")
+    VERSION=$(curl -sL https://github.com/kubevirt/containerized-data-importer/releases/latest | grep -m1 -o "v[0-9]\.[0-9]*\.[0-9]*")
```

Comment:
This seems to be the globally accepted way of finding the latest release of a github project...
However, it's effectively grepping the source of an entire webpage for a not-so-specific regex, which is not so reliable.
I mean, this is a random line from the source code, which comes *so close* to matching the regex:
```
<path fill-rule="evenodd" d="M0 6.75C0 5.784.784 5 1.75 5h1.5a.75.75 0 010 1.5h-1.5a.25.25 0 00-.25.25v7.5c0 .138.112.25.25.25h7.5a.25.25 0 00.25-.25v-1.5a.75.75 0 011.5 0v1.5A1.75 1.75 0 019.25 16h-7.5A1.75 1.75 0 010 14.25v-7.5z"></path><path fill-rule="evenodd" d="M5 1.75C5 .784 5.784 0 6.75 0h7.5C15.216 0 16 .784 16 1.75v7.5A1.75 1.75 0 0114.25 11h-7.5A1.75 1.75 0 015 9.25v-7.5zm1.75-.25a.25.25 0 00-.25.25v7.5c0 .138.112.25.25.25h7.5a.25.25 0 00.25-.25v-7.5a.25.25 0 00-.25-.25h-7.5z"></path>
```

Here's an alternative idea, let me know what you think!
```suggestion
    VERSION=$(curl -sL -o /dev/null -w %{url_effective} https://github.com/kubevirt/containerized-data-importer/releases/latest | xargs basename)
```